cmd/crc/cmd/config/view.go

```diff
@@ -56,6 +56,10 @@ func determineTemplate(tempFormat string) (*template.Template, error) {
 func runConfigView(cfg map[string]interface{}, tmpl *template.Template, writer io.Writer) error {
 	var lines []string
 	for k, v := range cfg {
+		// In case of pull secret don't show the value.
+		if k == "pullsecret" {
```

Comment:
should be a const value to make this more obvious and ability to lookup references